src/devtools/devtools.ts

```diff
@@ -3,7 +3,7 @@
  * on Chrome Development Tools window.
 */
 chrome.devtools.panels.create(
-  'Angular2',
+  'Augury',
```

Comment:
I don't think so, this was one of the feedback in usability study also. @igor-ka ?